src/playlist-loader.js

```diff
@@ -464,6 +484,11 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
           // MEDIA_ERR_NETWORK
           code: 2
         };
+        if (loader.state === 'HAVE_NOTHING') {
+          loader.started = false;
+        } else {
+          loader.state = startingState;
```

Comment:
Is something else changing the state necessitating a reversion to the starting state?